### ISSUE TITLE
"wake on keyboard" - new feature to Dasharo Workstation

### ISSUE THREAD

**Issue by rafkoch** [Dasharo/dasharo-issues#18]
It would be nice to turn on the computer by pressing a combination of keys on the keyboard (e.g. some password).

20 years ago, this was how I turned on my PC.  I used to set a POWER ON PASSWORD in my computer's BIOS.

Many laptops nowadays have a power button unavailable without opening the laptop's screen. I know users like me who use their laptops with a monitor but without docking stations and we have to open the screen only to start the computer and close the screen again after a while.

This feature "wake on keyboard"  would be very useful for us.

I know that the Dasharo Workstation is currently not available for laptops, but I still think that adding this feature (wake on keyboard) will be very useful, for example for people who don't want to bend under the desk to turn on the PC :-).

**Comment by rafkoch** [comment]
maybe better name for this feature will "power-on password / button combination" not "wake on keyboard" ?
Maybe it could be although a closed number of key combinations to power on PC, 
without pressing the button on the computer case?

**Comment by miczyg1** [comment]
The feasibility of this feature highly depends on board design and how the power is distributed when the machine is off. You probably would like to have a USB keyboard to power on the machine, but the USB controller may not even be powered when the machine is off, which can make it impossible to implement. I will have to investigate it deeper.

**Comment by rafkoch** [comment]
Yes USB port (not PS2) is now usually used standard for keyboards.
